### PR TITLE
Remove $arrCache from the insert tag hooks

### PR DESCRIPTION
**Pull request by ausi**

### CODE REVIEW

File: core-bundle/src/Resources/contao/library/Contao/InsertTags.php [hunk 1]
```diff
@@ -1181,7 +1181,7 @@ static function ($allowedTag)
 								foreach ($GLOBALS['TL_HOOKS']['insertTagFlags'] as $callback)
 								{
 									$this->import($callback[0]);
-									$varValue = $this->{$callback[0]}->{$callback[1]}($flag, $tag, $arrCache[$strTag], $flags, $blnCache, $tags, $arrCache, $_rit, $_cnt); // see #5806
+									$varValue = $this->{$callback[0]}->{$callback[1]}($flag, $tag, $arrCache[$strTag], $flags, $blnCache, $tags, array(), $_rit, $_cnt); // see #5806
```
Comment (ausi) [inline core-bundle/src/Resources/contao/library/Contao/InsertTags.php:1184]: Here `$arrCache[$strTag]` is not about some cache but actually the current value of the insert-tag.

**Comment by Toflar** [comment]
Do we need to document the break in UPGRADE.md?

**Comment by ausi** [comment]
> Do we need to document the break in UPGRADE.md?

done in 22e4195aff09af64a174bf4050bd04bc733f6161

File: UPGRADE.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 4.* to 5.0
 
+### replaceInsertTags and insertTagFlags hooks
+
+The `$cache` parameter (6th for `replaceInsertTags` and 7th for `insertTagFlags`) is no longer passed to the hooks,
+an empty array gets passed instead.
```
Comment (leofeyer) [inline UPGRADE.md]:
```suggestion
### Insert tag hooks

The `$cache` parameter is no longer passed to the `replaceInsertTags` and the `insertTagFlags` hook. An empty array is
passed instead.
```

File: UPGRADE.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 4.* to 5.0
 
+### Insert tag hooks
+
+The `$cache` parameter is no longer passed to the `replaceInsertTags` and the `insertTagFlags` hook. An empty array is
```
Comment (Toflar) [inline UPGRADE.md]:
```suggestion
The `$cache` parameter is no longer passed to the `replaceInsertTags` and the `insertTagFlags` hooks. An empty array is
```

Comment (leofeyer) [reply UPGRADE.md]: @ausi My bad. 🙈

**Review by m-vo — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**